helm/oauth2-proxy/ci/tpl-values.yaml

```diff
@@ -8,6 +8,8 @@ ingress:
   tls:
     - hosts:
         - "{{ $.Release.Name }}.local"
+  annotations:
+    nginx.ingress.kubernetes.io/default-backend: "{{ $.Release.Name }}-default-backend"
```

Comment:
Why this? You don't need to add this.

Reply:
I have to confess I don't fully understand how the tests work - I added this so I could see if the template is being resolved and to ensure the linter won't complain about the format.

If you can give me a hint on where to add a separate test case for this, then I'm happy to do so :-)

Reply:
@gtudan to test the chart we use ct.
Specifically this https://github.com/helm/chart-testing/blob/main/doc/ct_lint.md
> Charts may have multiple custom values files matching the glob pattern '*-values.yaml' in a directory named 'ci' in the root of the chart's directory. The chart is linted for each of these files. If no custom values file is present, the chart is linted with defaults.

Reply:
@gtudan, I have already approved the PR, but if you want to add to the tests all the cases you added in the PR, that would be great, so we have them all covered.

Reply:
Ok, I did it for you. 😄